test/e2e/watchcache/watchcache_enabled_test.go

```diff
@@ -53,18 +53,20 @@ func TestWatchCacheEnabledForCRD(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 	orgPath, _ := framework.NewOrganizationFixture(t, server)
+	// note that we schedule the workspace on the root shard because
+	// we need a direct and privileged access to it for downloading the metrics
 	wsPath, _ := framework.NewWorkspaceFixture(t, server, orgPath, framework.WithRootShard())
-	rootShardConfig := server.RootShardSystemMasterBaseConfig(t)
+	clusterConfig := server.BaseConfig(t)
```

Comment:
does this really change the URL? both `BaseConfig` and `RootShardSystemMasterBaseConfig` point to the base URL.

Reply:
yes, in a multi-shard env `BaseConfig` point to a kubeconfig for the front-proxy whereas `RootShardSystemMasterBaseConfig` points to a kubeconfig for the root shard :)

Reply:
in a single shard env it doesn't matter